python/cuml/test/test_mbsgd_classifier.py

```diff
@@ -34,6 +34,7 @@
 @pytest.mark.parametrize('column_info', [unit_param([20, 10]),
                          quality_param([100, 50]),
                          stress_param([1000, 500])])
+@pytest.mark.xfail(raises=AssertionError)
```

Comment:
Why this xfail?

Reply:
Had moved the xfail from the try and except block. It was initially added because of an issue that was opened in github. I went through the code, that issue has been resolved. Thank you for catching this.